restrictedsessions/middleware.py

```diff
@@ -80,7 +90,11 @@ def validate_ip(self, request, remote_ip):
             except AddrFormatError:
                 # session_network must be IPv4, but remote_ip is IPv6
                 return False
-        return remote_ip in session_network
+        is_ip_valid = remote_ip in session_network
```

Comment:
I think it would be nicer to have a single place where we log about failures, so can you take this one out of this PR? If you want to improve logging, have a look at https://github.com/erikr/django-restricted-sessions/pull/10.